src/Blocks/QueryLoopExtension.php

```diff
@@ -43,6 +43,29 @@ public static function registerFrontendQuery(): void
             'query_loop_block_query_vars',
             function ($query, $block) {
                 $blockQuery = $block->context['query'] ?? [];
+
+                $is_new_ia = !empty(planet4_get_option('new_ia'));
+
+                $query['post_type'] = ['page'];
+
+                // This applies only to Actions List block
+                if ($blockQuery['isCustom']) {
```

Comment:
I'm wondering if we need this extra query variable 🤔 In JS we use the [namespace attribute](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/block-editor/QueryLoopBlockExtension/index.js#L22), any chance we can access it here too?

Reply:
Looking at the `$block` params we receive, I don't see any related to `namespace` :/

Reply:
Remember that this hook is only related to the `query` but not the `block`.

Reply:
I understand but we do get the `block` parameter which according to the function docs is a `WP_Block` so its attributes should be accessible ([WP_Block docs](https://developer.wordpress.org/reference/classes/wp_block/))

Reply:
In this case the `$block` refers to `core/query-no-results` and `core/post-template` but not to the `planet4-blocks/actions-list`. But I'm still checking if there is another better approach.

Reply:
Weird, feels like it should be `core/query`

Reply:
This filter is being called for the inner blocks `core/query-no-results` and `core/post-template`, but you cannot access to `namespace` from `core/query`. That's why wee need to implement a custom `isCustom` flag.